packages/lib-utils/src/config.ts

```diff
@@ -39,3 +40,27 @@ export const getUtilsConfig = () => {
 
   return config;
 };
+
+let reduxStore: Store;
```

Comment:
We could make `reduxStore` part of `UtilsConfig` so we won't need another setter function, but we can do this later on.

Reply:
This would break your previous comment on exposing all of the UtilsConfig to the API of AppInitSDK

Reply:
Agreed. Will leave as is for now

Reply:
Agreed, sorry for confusion.